### DISCUSSION TITLE
applying inputText searchable on detailRow feature

### DISCUSSION THREAD

**Discussion by MuhammadTanvir** [Power-Components/livewire-powergrid#738] [Q&A]
Is there is any possibilty to make  inputText searchable on detailRow feature. For example, I have articles table which have parent and child category. Initially the table shows all parent categories and child categories will show when I click expand the specific parent category. 
on the datasource I fatch the parent articles and on the details row I loop through all the child categories
```
 public function datasource(): Builder
    {
        $articles = Article::query()->whereNull('kVaterArtikel')->with('child');
        return $articles;
    }
```
But child categories are not showing when I seach by name or other fileds. So, how can I search the child categories info?
Thank you

**Comment by luanfreitasdev** [comment]
I believe it is currently not possible. Maybe this can work if we use model scope in the filters

**Reply by MuhammadTanvir** [reply]
Thanks for the reply..it would be good if you add that feature in V3
